src/overlay.css

```diff
@@ -22,6 +22,7 @@
   top: 0;
   left: 0;
   background-color: rgba(0, 0, 0, 0.8);
+  outline: none;
```

Comment:
I'm not sure we can make this change, I think neuroglancer uses this for showing errors in code